pkgs/applications/virtualization/OVMF/default.nix

```
@@ -62,10 +68,21 @@ edk2.mkDerivation projectDscPath {
 
   dontPatchELF = true;
 
+  passthru.getEfiPrefix = platform:
```

You could do

```suggestion
  passthru.getEfiPrefix = { platform ? pkgs.stdenv.hostPlatform }:
```

but maybe that's over-engineered, if you only need

```nix
  passthru.efiPrefix = let platform = pkgs.stdenv.hostPlatform; in
```

I'm inclined to assume that if the user wants to set `platform`, they've already set `hostPlatform` for cross compilation.

Then you can also simplify to:

```nix
  passthru = { firmware = ...; variables = ...; };
```

I am completely unfamiliar with this package though, so I don't really know.

You'd be free to add `getEfiPrefix` later if there's a real need.

---

Right, I think it's definitely better to go with your suggestion. We can always do more complicated stuff later.

---

Done.